tests/phpunit/BatchTest.php

```diff
@@ -17,7 +17,6 @@
  */
```

Comment:
@eileenmcnaughton why are you moving this file? i'm not sure if it will then be picked up because the class name doesn't match the file name then.

Reply:
ug that was an opps / ide trying to fix up psr issues

Reply:
@seamuslee001 fixed